content/docs/05.developer-guide/03.scripts.md

```diff
@@ -0,0 +1,800 @@
+---
+title: Python, R, Node.js and Shell Scripts
+---
+
+The [script plugin](https://github.com/kestra-io/plugin-scripts) allows you to orchestrate custom business logic written in `Python`, `R`, `Node.js`, `Powershell` and `Shell` scripts. By default, all these tasks run in individual Docker containers. You can optionally run those scripts in a local process or run them on a remote [worker group](../08.architecture.md#worker-group-ee). 
+
+
+## The `Commands` and `Script` tasks
+
+For each of these languages, Kestra provides two types of tasks: inline `Script` tasks, and `Commands` tasks triggered from the terminal based on the provided Docker image and optionally also based on a Git repository containing your code. 
```

Comment:
that was the purpose of the further sentence -- inline script vs Commands triggered from CLI. The example like python hello.py is in the table below

Reply:
actually no, I trimmed the table because docs couldn't render it, it was too wide 🙈


|Language|Default image|beforeCommands example|Script example|Commands example|Outputs|Metrics|Task type (full class name)|
|---|---|---|---|---|---|---|---|
|Python|python|pip install requests kestra|print("Hello World!")|python [hello.py](http://hello.py)|Kestra.outputs({'data': data})|Kestra.counter('nr_rows', len(df))|io.kestra.plugin.scripts.python.Commands/Script|
|R|r-base|Rscript -e "install.packages('dplyr')"|print("Hello World!")|Rscript hello.R|TBD|TBD|io.kestra.plugin.scripts.r.Commands/Script|
|Node.js|node|npm install json2csv|console.log('Hello World!');|node hello.js|Kestra.outputs({'status': d.status, 'text': d.data})})|TBD|io.kestra.plugin.scripts.node.Commands/Script|
|Shell|ubuntu|apt-get install curl|echo "Hello World!"|./hello.bash|echo '::{"outputs":{"nr_rows":800}}::’|echo '::{"metrics":[{"name":"nr_rows","type":"counter","value":800,"tags":{"schema":"analytics"}}]}::’|io.kestra.plugin.scripts.shell.Commands/Script|
|Powershell|[http://mcr.microsoft.com/powershell](http://mcr.microsoft.com/powershell)|Install-Module -Name ImportExcel|Write-Output "Hello World!"|.\hello.ps1|TBD|TBD|io.kestra.plugin.scripts.powershell.Commands/Script|

Reply:
Back to the sentence ;)
I didn't understand it and you talk about Docker and Git now as it's explained later I think a more light sentence would b better.

> For each of these languages, Kestra provides two types of tasks: inline `Script` tasks, and `Commands` tasks that allow executing a list of commands by a script interpreter.

Reply:
that's a good way of putting it, adding verbatim